### ISSUE TITLE
Proposed feature: XY map in psfStarChooser window

### ISSUE THREAD

**Issue by Mikea1985** [fraserw/trippy#35]
I just thought of this and wanted to write it down so I don't forget.

What: Add a new subplot to the psfStarChooser window which shows the X&Y image co-ordinates of the stars. 
Why: Useful for cases where the psf varies significantly over the field (due to large field of view, being near edge of field, or image being mosaic of multiple images) so that the user can easily identify stars that are closest to the source of interest. 
How: The current window is divided into a 4x4 grid with some subplots spanning multiple grid fields. I'd suggest increasing this to 5 in height, shifting the radial profile down one step, sticking the image cutout in the little hole created above the radial profile, and then using the lower right corner (which should be 3x2 grid fields) for the plot of the co-ordinates. Alternatively (if that seems like it's getting too cluttered), we could drop the histograms (I don't find them useful, but I don't know whether this opinion is shared), and stick with the existing layout with the XY subplot added in the lower right corner. 
Who: I can do it (although input on favoured final appearance is appreciated), just not until after new year.

**Comment by Mikea1985** [comment]
Bonus idea: If the points were colour coded in some way in both the X vs Y and RMS vs FWHM plot (in which case we'd have to switch from red/blue to using symbol shapes to denote which are selected/unselected), it would be super easy to notice if there is a gradient in the PSF width/quality across the field. But this might be making things more complicated than necessary.

**Comment by fraserw** [comment]
Yeah I have actually implemented this in trippy v0. Wasn't hard. But at the time I had fields with few stars, which wasn't so informative. Moreover, the science target location wasn't displayed, as that's not an option one passes to psfStarChooser. So I determined that interface simplicity won over the minor improvement in utility. Colour coding based on different things was also pretty easy.

That being said, we can reimplement this, if you think the above limitations aren't too limiting.

On a side note, can you email me an image with a highly spatially variable PSF? I have a bunch of data from a bunch of different cameras (Megacam, SC, HSC, GMOS, etc.) and have never noticed much in the ways of PSF spatial variability.
